<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="hxnb-fcoronel" name="Default test" tests="11" failures="1" timestamp="5 Dec 2016 16:12:01 GMT" time="368.859" errors="0">
  <testcase name="launchSite" time="129.297" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testChandeliersList" time="36.334" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavMenu" time="1.44" classname="com.howtotestngdo.test.FirstTest">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Chandeliers] but found [Chandelier]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
	expected [Chandeliers] but found [Chandelier]
	at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:43)
	at com.howtotestngdo.test.FirstTest.testNavMenu(FirstTest.java:122)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:100)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:811)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1137)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:753)
	at org.testng.TestRunner.run(TestRunner.java:607)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:368)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:363)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:321)
	at org.testng.SuiteRunner.run(SuiteRunner.java:270)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1284)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1209)
	at org.testng.TestNG.runSuites(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1096)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:236)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:81)
]]>
    </failure>
  </testcase> <!-- testNavMenu -->
  <testcase name="testNavigationCeilingLights" time="28.518" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationChandeliers" time="4.472" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationFans" time="8.016" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationFurniture" time="4.11" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationHomeDecor" time="11.676" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationLamps" time="64.006" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationOutdoor" time="5.839" classname="com.howtotestngdo.test.FirstTest"/>
  <testcase name="testNavigationWallLights" time="6.072" classname="com.howtotestngdo.test.FirstTest"/>
</testsuite> <!-- Default test -->
